R/mpf_exporters.R

```diff
@@ -426,20 +437,21 @@ export_as_rtf <- function(x,
     names(margins) <- marg_order
   }
 
-  fullmf <- matrix_form(x, indent_rownames = TRUE, fontspec = fontspec)
-  req_ncols <- ncol(fullmf) + as.numeric(mf_has_rlabels(fullmf))
-  if (!is.null(colwidths) && length(colwidths) != req_ncols) {
-    stop(
-      "non-null colwidths argument must have length ncol(x) (+ 1 if row labels are present) [",
-      req_ncols, "], got length ", length(colwidths)
-    )
-  }
+  # NEEDS TO BE INTO paginate_to_mpfs so to have this check once for all paginations
+  # fullmf <- matrix_form(x, indent_rownames = TRUE, fontspec = fontspec)
```

Comment:
we have a check inside `paginate_to_mpfs` that does this check so for all exports